app/manager-frontend/public/locales/dev/core.json

```
@@ -71,7 +71,8 @@
     },
     "feedback": {
       "loginFailed": "We couldn't log you in with the username and password you provided.",
-      "passwordResetInfo": "Please contact your system admin to reset your password"
+      "passwordResetInfo": "Please contact your system admin to reset your password",
+      "unconfirmedAccountInfo": "Please contact your system admin to activate your account"
```

```suggestion
      "passwordResetInfo": "Please contact your system admin to reset your password.",
      "unconfirmedAccountInfo": "Please contact your system admin to activate your account."
```
⛏️ The top one has a full-stop, so just for the sake of consistency